recipe/migrations/arch_rebuild.txt

```diff
@@ -34,6 +34,7 @@ pytest-benchmark
 conda-forge-ci-setup
 oclgrind
 openmpi
+ucx
```

Comment:
This should be the feedstock name right?

Reply:
Ah sorry thought these were package names

Reply:
Fixing with PR ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/2657 )